tqecjs/src/App.js

```diff
@@ -2,6 +2,8 @@ import './App.css';
 import { Stage } from '@pixi/react';
 import PixiApp from './workspace';
 
+const tqecBlue = 0x1099bb;
```

Comment:
Maybe we can make css files for this? I

Reply:
Agreed. The only real snag is that the React Stage expects the `options` element as a JavaScript object, not a CSS file.

Opening an issue to implement this later on.